src/daemon/ca/certauth.rs

```diff
@@ -964,6 +973,123 @@ impl CertAuth {
         Ok(res)
     }
 
+    // Suspend a child. The intention is that this is called when it is discovered
+    // that the child has been inactive, i.e. not contacting this parent for a pro-longed
+    // period of time (hours).
+    //
+    // When a child is suspended we need to:
+    // - mark it as suspended
+    // - withdraw all certificates issued to it (suspend them)
+    fn child_suspend_inactive(&self, child_handle: &ChildHandle) -> KrillResult<Vec<CaEvt>> {
```

Comment:
This fn doesn't care about whether the child is inactive or not right, it just suspends the given child, so perhaps remove inactive from the fn name?